nextflow/modules/prodigal.nf

```diff
@@ -13,6 +13,7 @@ process prodigal {
     
     script:
     """
+    sed -i "s/ /|/" ${fasta}
```

Comment:
This one is repeated in several places, it would be better to do that in a single place.. in the rename step probably

Reply:
agree

Reply:
Not sure when rename step is happening, but we need to rename the prophage sequence IDs after the viral predictions are done.

Reply:
The problem comes from our parse_viral_predictions script... see my previuos comment